### PR TITLE
Add rule for RHEL-08-020090

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

Added new rule for RHEL-08-020090. No automated remediation due to the custom nature of the settings. 

#### Rationale:
STIG Reference: https://vaulted.io/library/disa-stigs-srgs/red_hat_enterprise_linux_8_security_technical_implementation_guide/V-230355

### CODE REVIEW

File: linux_os/guide/services/sssd/sssd_enable_certmap/oval/shared.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<def-group>
+      <definition class="compliance" id="{{{ rule_id }}}"  version="1">
+          {{{ oval_metadata("SSD should be configured to authenticated identity to the user or group account
```
Comment (cipherboy) [inline linux_os/guide/services/sssd/sssd_enable_certmap/oval/shared.xml:3]: SSSD :-)

**Comment by jan-cerny** [comment]
@mildas Have you seen the fail of the content test filtering and SSGTS?

File: linux_os/guide/services/sssd/sssd_enable_certmap/oval/shared.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    <ind:textfilecontent54_test check="all" check_existence="all_exist"
+                                comment="check if certmap is enabled in /etc/sssd/sssd.conf"
+                                id="test_{{{rule_id}}}_section" version="1">
+        <ind:object object_ref="obj_{{{rule_id}}}_section" />
+    </ind:textfilecontent54_test>
+
+     <ind:textfilecontent54_object id="obj_{{{rule_id}}}_section" version="1">
+         <ind:filepath>/etc/sssd/sssd.conf</ind:filepath>
+         <ind:pattern operation="pattern match">^\[certmap\/.+\/.+\]$</ind:pattern>
+         <ind:instance datatype="int" operation="equals">1</ind:instance>
```
Comment (jan-cerny) [inline linux_os/guide/services/sssd/sssd_enable_certmap/oval/shared.xml:19]: I think you should use template lineinfile instead. It;s possible to generate only OVAL and turn off the Ansible and Bash:
```
template:
    name: lineinfile
    vars:
      path: '/etc/sssd/sssd.conf'
      text: '^\[certmap\/.+\/.+\]$'
    backends:
        ansible: "off"
        bash: "off"
```

File: linux_os/guide/services/sssd/sssd_enable_certmap/tests/default.fail.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
```
Comment (jan-cerny) [inline linux_os/guide/services/sssd/sssd_enable_certmap/tests/default.fail.sh:1]:
```suggestion
#!/bin/bash
```

File: linux_os/guide/services/sssd/sssd_enable_certmap/tests/with_section.pass.sh [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
```
Comment (jan-cerny) [inline linux_os/guide/services/sssd/sssd_enable_certmap/tests/with_section.pass.sh:1]:
```suggestion
#!/bin/bash
```

**Comment by ggbecker** [comment]
> @mildas Have you seen the fail of the content test filtering and SSGTS?

this should be solved by https://github.com/ComplianceAsCode/content/pull/7305

he just needs to rebase

File: linux_os/guide/services/sssd/sssd_enable_certmap/oval/shared.xml (hunk 1 above)
Comment (Mab879) [reply linux_os/guide/services/sssd/sssd_enable_certmap/oval/shared.xml]: Mooted, no longer using custom OVAL. :)

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**